<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="asda1716po02925" name="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest" tests="10" failures="3" timestamp="28 Jun 2017 11:18:45 GMT" time="0.053" errors="0">
  <testcase name="getListOfTeams_should_return_a_list_if_Team_object_from_db_json" time="0.006" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&lt;[3]&gt; but was:&lt;[0]&gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[3]> but was:<[0]>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getListOfTeams_should_return_a_list_if_Team_object_from_db_json(TeamsJsonReaderTest.java:103)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- getListOfTeams_should_return_a_list_if_Team_object_from_db_json -->
  <testcase name="getListOfActiveIndividuals_ahould_return_array_list_of_only_active_individual_objects" time="0.001" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest"/>
  <testcase name="Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive" time="0.004" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&lt;[1]&gt; but was:&lt;[0]&gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[1]> but was:<[0]>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive(TeamsJsonReaderTest.java:122)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive -->
  <testcase name="getListOfInactiveIndividuals_should_return_array_list_of_only_inactive_individual_objects" time="0.004" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest"/>
  <testcase name="getIndividualByName_should_return_Individual_object_matching_name" time="0.002" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest"/>
  <testcase name="getIndividualById_should_return_Individual_object_matching_id" time="0.003" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest"/>
  <testcase name="getIndividualById_should_throw_ObjectNotFoundException_for_incorrect_id" time="0.001" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest"/>
  <testcase name="getIndividualByName_should_throw_ObjectNotFoundException_for_incorrect_name" time="0.001" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest"/>
  <testcase name="Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active" time="0.029" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&lt;[2]&gt; but was:&lt;[0]&gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[2]> but was:<[0]>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active(TeamsJsonReaderTest.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active -->
  <testcase name="getListOfIndividuals_should_return_array_list_of_all_individual_objects" time="0.002" classname="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest"/>
</testsuite> <!-- com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest -->
